README.md

```diff
@@ -18,3 +18,13 @@ Run tests using Cypress GUI:
 ```
 npx cypress open
 ```
+
+## Setup Test Data
+
+```
+wp eval-file bin/e2e-tests-setup.php --user=<admin-id>
```

Comment:
might be worth noting that <admin-id> must be a numeric ID.

Reply:
`--user` is global WP CLI parameter and I think it accepts numeric ID | login | email.

Reply:
Oh my bad, I didn't know that WP-CLI was handling ithese.